lib/bitstyles_phoenix/helper/classnames.ex

```diff
@@ -41,65 +45,48 @@ defmodule BitstylesPhoenix.Classnames do
       "foo"
 
       iex> classnames({"foo", 1 == 2})
-      ""
+      false
 
       iex> classnames(["  foo  boing  ", {"bar", 1 == 2}, :baz])
       "foo boing baz"
   """
-  def classnames(arg) do
+  def classnames(arg, opts \\ [backwards_compatible: true]) do
     arg
     |> List.wrap()
     |> Enum.map(&normalize/1)
     |> Enum.flat_map(&split/1)
     |> Enum.reject(&remove_class?/1)
     |> Enum.uniq()
+    |> Enum.map(&bitstyles_version(&1, opts))
     |> Enum.join(" ")
+    |> case do
+      "" -> false
```

Comment:
🤔  I guess you did this because in HEEx `<div class={false} />` renders a `<div />`, right? Which is quite nice, but now this renders from EEx `<div class="<%= false %>" />` to `<div class="false" />`. Is that a problem? Do we care about the `false` class? Not adding the class attribute if it's not needed is definitely a win.

Reply:
I can add a breaking change comment about it. I think we do not need to support the EEX case tbh.